recipe/meta.yaml

```diff
@@ -130,17 +131,20 @@ outputs:
       description: |
         Intel Math Kernel Library is a BLAS implementation tuned for high performance on Intel CPUs.
         This package is a repackaged set of binaries obtained directly from Intel\'s anaconda.org channel.
-      doc_url: https://software.intel.com/en-us/mkl/documentation
+      doc_url: https://www.intel.com/content/www/us/en/developer/tools/oneapi/onemkl.html
     test:
       commands:
         - ls -A $PREFIX/lib/*  # [unix]
+        - test -f $PREFIX/lib/libmkl_core.so.2  # [unix]
```

Comment:
I wonder if we can make this test more flexible so that it wont start failing once MKL bumps the SO version.

Reply:
i was thinking about this - there is no good alternative other than globbing.  We probably can keep this as is 

In upcoming 2024 release oneDAL will go from .1 to .2, so this would require tests update during upgrade to 2024.0
@Alexsandruss